community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -129,6 +129,10 @@
     public static final Setting<Long> transaction_start_timeout =
             setting( "transaction_start_timeout", DURATION, "1s" );
 
+    @Description("The location of the internal diagnostics log.")
+    @Internal
```

Comment:
Current setting is marked with `@Internal`  and also contains _internal_ as part of the name. In the same time already closest neighbour `store_internal_log_rotation_threshold` do not follow this pattern. Let's mark all of them that matched to the principle?